repositories.yaml

```diff
@@ -546,3 +546,23 @@ applications:
     channels:
       - v1_name: bergamot
         app_id: org.mozilla.bergamot
+
+  - app_name: rally_debug
+    canonical_app_name: Rally Core Add-on Debug Ingestion
+    app_description: Rally Core Add-on debug ingestion
+    url: https://github.com/mozilla-rally/rally-core-addon
+    notification_emails:
+      - than@mozilla.com
+    branch: master
+    metrics_files:
+      - metrics.yaml
+    ping_files:
+      - pings.yaml
+    dependencies:
+      - glean-js
+    retention_days: 180
+    channels:
+      - v1_name: rally
+        app_id: rally.debug
+    encryption:
+      use_jwk: true
```

Comment:
Tests are currently failing because this is a new field here that we haven't built support for yet.

We'll have to add this to the JSON schema in `probeinfo_api.yaml` for `ApplicationYaml`. I don't have time before EOD to look back through the proposals and ensure this is the right thing to do. The pipeline is expecting the JSON schema for Rally pings to include `jwe_mappings` in `mozPipelineMetadata`. Did we end up deciding that we'd use probe-scaper metadata to configure this injection?  If so, changes will be needed in mozilla-schema-generator to pick up this field and respond to it, and we'll need to make it available in the probeinfo API.

Reply:
I see now that you gave a link to the relevant bit of the proposal. So yes, this is the right thing to do :) but we'll need to make some additional code changes here to support it. I can prioritize working on this on Monday.